lips/lip-6.mdx

```diff
@@ -101,35 +105,35 @@ The following transaction scripts can be run by the Validator Owner:
 
 ## Validator Owner to set the Validator Operator
-Validator Owner may call a transaction script `set_validator_operator` to set the `operator_account` field of the `ValidatorConfig` resource, it will succeed only if the `operator_account` has a Validator Operator role.
+Validator Owner may call a transaction script `set_validator_operator` to set the `operator_account` field of the `ValidatorConfig` resource. This script will succeed only if the `operator_account` has a Validator Operator role.
 
-The script takes a human name of the Validator Operator (for extra validation) and its on-chain address. When successful this transaction changes the field `operator_account` of the `ValdiatorConfig` resource of the Validator Owner calling the transaction. When the `operator_account` is set, only the `operator_account` may change the `config` field of the `ValidatorConfig` resource.
+The transaction script takes a human-readable name of the Validator Operator (for an additional validation check) and its on-chain address. When successful this transaction changes the field `operator_account` of the `ValdiatorConfig` resource of the Validator Owner calling the transaction. When the `operator_account` is set, only the `operator_account` may change the `config` field of the `ValidatorConfig` resource.
 
 ## Validator Operator (or Owner) to set the Config
-For a Validator Owner who is not yet added to the Validator Set, the Validator Operator (or Validator Owner if `operator_account` is unset) may change the `config` by calling a transaction script `register_validator_config`, it takes an address of the Validator Owner (to avoid confusion when one Validator Operator runs Validator Nodes for multiple Validator Owners), a new consensus public key (a 32 elements byte array) and two byte arrays for network addresses: `validator_network_addresses` for other validators to communicate with this validator during consensus and `fullnode_network_addresses` for full-nodes, clients or validators who are catching-up.
+For a Validator Owner who is not yet added to the Validator Set, the Validator Operator (or Validator Owner if `operator_account` is unset) may change the `config` by calling a transaction script `register_validator_config`. This script takes an address of the Validator Owner (to avoid confusion when one Validator Operator runs Validator Nodes for multiple Validator Owners), a new consensus public key (a 32 elements byte array) and two byte arrays for network addresses: `validator_network_addresses` for other Validator Operators to communicate with this Validator Operator during consensus and `fullnode_network_addresses` for Full Nodes, clients or validators who are catching-up.
```

Comment:
"fullnode_network_addresses" for network participation for nodes outside the current Validator Set.